shell/platform/android/io/flutter/view/FlutterView.java

```diff
@@ -542,17 +542,62 @@ protected void onSizeChanged(int width, int height, int oldWidth, int oldHeight)
         super.onSizeChanged(width, height, oldWidth, oldHeight);
     }
 
+    // Decide if we want to zero the sides to remove padding for hidden navbar.
+    enum ZeroSides { NONE, LEFT, RIGHT, BOTH }
+    ZeroSides calculateShouldZeroSides(WindowInsets insets) {
```

Comment:
I assume we'll need to update this code to handle the new cutout APIs. If that's the case, please add a TODO to help out the future implementor while you're here.